src/parser/tests/parser-csv.test.ts

```diff
@@ -0,0 +1,45 @@
+// node
+import { readFile } from 'node:fs/promises';
+import path from 'node:path';
+
+// libs
+import { expect, test } from 'vitest';
+
+// test functions
+import { parseStream } from '../parser.ee.js';
+
+const csvPath = path.join(import.meta.dirname, 'sampleFiles', 'test.csv');
+// eslint-disable-next-line security/detect-non-literal-fs-filename
```

Comment:
be careful with that line.

nothing wrong here, but I want you to be aware that disabling eslint should be an exception, and carefully considered, especially security rules


I will disable this linter rule on test files, as this security issue will never arise on test files